service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java

```diff
@@ -113,4 +114,9 @@ public void run() {
       srClient.checkIsolationAddressAvailable();
     }
   }
+
+  @VisibleForTesting
+  public void setAddrCheckInit(boolean addrCheckInit) {
```

Comment:
It's not recommended to expose the hiden property for UT purpose. If this method is only used for `TestMicroserviceInstanceRegisterTask`, you can delete the `public` modifier. Just exposing `isAddrCheckInit` property to the classes under the same package seems enough.

Reply:
fixed